tree/dataframe/test/CMakeLists.txt

```diff
@@ -38,7 +38,9 @@ if (imt)
 endif()
 
 ROOT_ADD_GTEST(datasource_more datasource_more.cxx LIBRARIES ROOTDataFrame)
-ROOT_ADD_GTEST(datasource_root datasource_root.cxx LIBRARIES ROOTDataFrame)
+# TODO: RRootDS is in the process of being hidden from users, and it's currently deprecated.
+# Re-enable these tests after moving RRootDS to the internal namespace
+# ROOT_ADD_GTEST(datasource_root datasource_root.cxx LIBRARIES ROOTDataFrame)
```

Comment:
Is there a ticket so this doesn't get lost?

Reply:
good point, this is now https://sft.its.cern.ch/jira/browse/ROOT-10741